SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -113,6 +113,7 @@ private void submitRequest(SingularityRequest request, Optional<SingularityReque
 
     if (oldRequest.isPresent()) {
       authorizationHelper.checkForAuthorization(oldRequest.get(), user, SingularityAuthorizationScope.WRITE);
+      authorizationHelper.checkForAuthorizedChanges(request, oldRequest.get(), user.get()); // previous check guarantees that user is present
```

Comment:
So in this situation, if no user is present, do we fail since they _technically_ are not in the request group?